backend/src/main/resources/application.yaml

```diff
@@ -0,0 +1,39 @@
+spring:
+  application:
+    name: backend
+  ssl:
+    bundle:
+      jks:
+        server:
+          key:
+            alias: "igc-dev"
+          keystore:
+            location: "classpath:ssl/igc-dev.p12"
+            password: "password"
+            type: "PKCS12"
+
+server:
+  port: 8080
+  ssl:
+    bundle: "server"
+    enabled-protocols: "TLSv1.3"
+
+management:
+  server:
+    port: 9090
+  endpoints:
+    web:
+      exposure:
+        include: "openapi, swagger-ui"
+
+springdoc:
+  show-actuator: true
+  use-management-port: true
+
+inspectit:
+  gepard:
+    security:
+      cors:
+        path-pattern: "/**"
+        allowed-origins: "*"
+        allowed-methods: "*"
```

Comment:
I believe it would make things easier, if we stay close to the current inspectit-config-server configuration.
It will definitely reduce the migration effort :)

So we might use `inspectit-config-server` as our "root-property". Don't know if we necessarily need the `gepard` property.

The current configuration model for reference:
- https://github.com/inspectIT/inspectit-ocelot/blob/master/components/inspectit-ocelot-configurationserver/src/main/java/rocks/inspectit/ocelot/config/model/InspectitServerSettings.java
- https://github.com/inspectIT/inspectit-ocelot/blob/master/components/inspectit-ocelot-configurationserver/src/main/resources/application.yml

Reply:
Done.